circuits/circom/.gitignore

```diff
@@ -0,0 +1,7 @@
+# circom/snarkjs artifacts
+*.r1cs
+*.sym
```

Comment:
circom-tester should be putting artifacts in temporary folder and cleaning it up again. Did you find some artifacts were being generated? Or was it just when you ran `circom` manually for testing?

Reply:
While testing, I actually stopped the script/process before it ended with ctrl + c. Maybe that's why these artifacts weren't cleaned up.